src/data/quot.lean

```diff
@@ -262,10 +263,22 @@ noncomputable def quotient.choice {ι : Type*} {α : ι → Type*} [S : Π i, se
   (f : Π i, quotient (S i)) : @quotient (Π i, α i) (by apply_instance) :=
 ⟦λ i, (f i).out⟧
 
-theorem quotient.choice_eq {ι : Type*} {α : ι → Type*} [Π i, setoid (α i)]
+@[simp] theorem quotient.choice_eq {ι : Type*} {α : ι → Type*} [Π i, setoid (α i)]
   (f : Π i, α i) : quotient.choice (λ i, ⟦f i⟧) = ⟦f⟧ :=
 quotient.sound $ λ i, quotient.mk_out _
 
+
+attribute [elab_as_eliminator]
+lemma quotient.induction_on_pi
+   {ι : Type*} {α : ι → Sort*} [s : ∀ i, setoid (α i)]
+   {φ : (Π i, quotient (s i)) → Prop} (q : Π i, quotient (s i))
+   (h : ∀ a : Π i, α i, φ (λ i, ⟦a i⟧)) : φ q :=
+begin
+  rw ← (funext (λ i, quotient.out_eq (q i)) : (λ i,  ⟦(q i).out⟧) = q),
```

Comment:
It looks like these variable names are more in line with the rest of `mathlib`.
```suggestion
@[elab_as_eliminator] lemma quotient.induction_on_pi
   {ι : Type*} {α : ι → Sort*} [s : ∀ i, setoid (α i)]
   {p : (Π i, quotient (s i)) → Prop} (f : Π i, quotient (s i))
   (h : ∀ a : Π i, α i, p (λ i, ⟦a i⟧)) : φ f :=
begin
  rw ← (funext (λ i, quotient.out_eq (f i)) : (λ i,  ⟦(f i).out⟧) = f),
```

Reply:
I was trying to imitate the style of the other `induction` lemmas. For example, induction_on₂ has:

```lean
attribute [elab_as_eliminator]
protected lemma induction_on₂
   {φ : quotient s₁ → quotient s₂ → Prop} (q₁ : quotient s₁) (q₂ : quotient s₂) (h : ∀ a b, φ ⟦a⟧ ⟦b⟧) : φ q₁ q₂ :=
quotient.ind (λ a₁, quotient.ind (λ a₂, h a₁ a₂) q₂) q₁
```

I'm fine with the new names too though.